tileservercache_templates/poracle-gym.json

```diff
@@ -11,6 +11,7 @@
         #for(stop in nearbyStops):
         {
             "url": "#if(stop.imgUrl):#(stop.imgUrl)#else:#(uiconPokestopUrl)#endif",
+            "fallback_url": "https://raw.githubusercontent.com/KartulUdus/PoracleJS/images/fallback/stop.png",
```

Comment:
This may be better as (untested)
`#if(stop.type=='gym'):https://raw.githubusercontent.com/KartulUdus/PoracleJS/images/fallback/gym.png#else:https://raw.githubusercontent.com/KartulUdus/PoracleJS/images/fallback/stop.png#endif`